src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -3619,20 +3635,24 @@ public void testNetworksConnectContainerWithEndpointConfig() throws Exception {
     assertThat(network.containers().size(), equalTo(1));
     assertThat(networkContainer, notNullValue());
     final ContainerInfo containerInfo = sut.inspectContainer(containerCreation.id());
+    assertNotNull(containerInfo.networkSettings().networks());
     assertThat(containerInfo.networkSettings().networks().size(), is(2));
+    assertThat(containerInfo.networkSettings().networks(), hasKey(networkName));
     final AttachedNetwork attachedNetwork =
             containerInfo.networkSettings().networks().get(networkName);
-    assertThat(attachedNetwork, is(notNullValue()));
```

Comment:
Good question. I hadn't really thought about why to use `assertThat` vs other various `assertFoo`s. I'll change those back.